pom.xml

```diff
@@ -17,15 +17,14 @@
   <url>https://github.com/jenkinsci/p4-plugin</url>
 
   <properties>
-    <jenkins.version>2.346.3</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.375.3</jenkins.version>
```

Comment:
As per our Jenkins version policy, we only can update to 2.361.4.
Also we need java.level, it will make sure Maven builds using Java 11, if some other version of java is available on build machines.

Reply:
A significant number of users depend on the P4 plugin with Jenkins versions predating 2.361.4. Expanding support beyond this version would impact roughly 50% of our user base. Our objective is to offer support for Jenkins versions released within the last two years, unless there is a compelling need to do otherwise.